docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/stepper/accessibility/
+---
+
+## Accessibility
+
+The Stepper component has been designed with accessibility in mind. It can be used with keyboard navigation and includes properties that enhance the experience for users of assistive technologies.
```

Comment:
I think there are still some repeating sentences in this part (and under the list of props) 😬 They say the same thing in different words. Maybe it can be further reduced to not cause clutter?